piksi_tools/console/tracking_view.py

```diff
@@ -99,7 +101,7 @@ def update_plot(self):
       if self.states[n].state == 0:
         plot_labels.append('Ch %02d (Disabled)' % n)
       else:
-        plot_labels.append('Ch %02d (PRN%02d)' % (n, self.states[n].prn+1))
+        plot_labels.append('Ch %02d (PRN%02d)' % (n, self.states[n].prn))
     plots = dict(zip(plot_labels, self.plots))
     self.plot.legend.plots = plots
```

Comment:
Note that there's also: 

- https://github.com/swift-nav/piksi_tools/blob/e674ea9457affce3ad8b34dbce05f2e8c56b5f59/piksi_tools/console/observation_view.py#L113
- https://github.com/swift-nav/piksi_tools/blob/e674ea9457affce3ad8b34dbce05f2e8c56b5f59/piksi_tools/console/observation_view.py#L191

... and hopefully nothing else.